app/assets/javascripts/discourse/controllers/preferences/second-factor-backup.js.es6

```diff
@@ -88,7 +88,9 @@ export default Ember.Controller.extend({
             errorMessage: null,
             backupCodes: response.backup_codes
           });
-          model.set("second_factor_backup_enabled", true);
+
+          this.set("backupEnabled", true);
+          this.set("remainingCodes", 10);
```

Comment:
We should probably do a length count on `response.backup_codes` instead of hard coding the value here. Also, you can do the following when setting multiple properties at once 

```
this.setProperties({
  backupEnabled: true,
  remainingCodes: response.backup_codes.length
})
```

Reply:
Ah true, thank you!